multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -254,3 +259,53 @@ func (c *Cluster) Destroy() error {
 	_, err := cfSvc.DeleteStack(dreq)
 	return err
 }
+
+func (c *Cluster) validateDNSSettings() error {
+	if !c.CreateRecordSet {
+		return nil
+	}
+
+	if c.RecordSetTTL < 1 {
+		return fmt.Errorf("TTL must be at least 1 second")
+	}
+
+	if c.HostedZone == "" {
+		return fmt.Errorf("hostName cannot be blank when createRecordSet is true")
+	}
+
+	r53 := route53.New(c.session)
```

Comment:
`validateDNSSettings` should take the route53 service as a parameter. As is done with `validateExistingVPC`, an interface should be stubbed out and used to exercise this codepath in the  cluster unit tests.